test/test-pages/daringfireball-1/expected-metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Daring Fireball: Colophon",
+  "byline": null,
```

Comment:
Actually not, as there's no semantic markup in this page stating who the author is.